README.md

```diff
@@ -29,6 +29,7 @@ Do you want to add your own? Please feel free to open a pull request!
 | **Tripsta** | Trip tracking app with statistics | 50+ (Google Play) | [<img src="assets/website.png"/>](https://tripsta.app/) [<img src="assets/android.png"/>](https://play.google.com/store/apps/details?id=com.universley.tripsta&hl=en) [<img src="assets/ios.png"/>](https://apps.apple.com/us/app/tripsta/id6448712233?l=en-US) |
 | **TV Show & Movie Tracker -Trakt** | TV Show Tracker let you discover, track and comment your favorite TV shows & movies. | 100K+ (Google Play) | [<img src="assets/android.png"/>](https://play.google.com/store/apps/details?id=com.jonathanantoine.TVST&hl=en) [<img src="assets/ios.png"/>](https://apps.apple.com/nl/app/tv-show-tracker-trakt-client/id1228141510?l=en-US) |
 | **UnityPay: Joint Expenses** | Expense Tracker for Couples | 100+ (Google Play) | [<img src="assets/website.png"/>](https://www.williambelle.co/apps/unity-pay) [<img src="assets/android.png"/>](https://play.google.com/store/apps/details?id=co.williambelle.unitypay&hl=en&hl=en) [<img src="assets/ios.png"/>](https://apps.apple.com/us/app/tripsta/id6448712233?l=en-US) |
+| **TopBeers** | Collect and discover new beers | 50+ (Google Play) | [<img src="assets/website.png"/>](https://tripsta.app/) [<img src="assets/android.png"/>](https://play.google.com/store/apps/details?id=com.companyname.topbeers.app&pli=1) [<img src="assets/ios.png"/>](https://apps.apple.com/es/app/topbeers/id6502869257) |
```

Comment:
The website says Tripsta? 😄 do you have a website or do you want to remote it?

Reply:
Also would it be possible to link to the English stores? Check the other links on the query parameter. That way I think most people will be able to understand what it says at least